src/lwc/psElevateTokenHandler/__tests__/psElevateTokenHandlerTest.js

```diff
@@ -0,0 +1,105 @@
+import psElevateTokenHandler from 'c/psElevateTokenHandler';
+
+const createPostMessageEvent = (type) => {
+   switch (type) {
+      case 'valid' :
+         return {
+            origin : "https://flow-connect-2738-dev-ed--c.visualforce.com",
+            data : {
+               "isLoaded":true
+            }
+         };
+      case 'invalid' :
```

Comment:
What do you think about having one of these testing scenarios mimic the case where the origin is expected but the data property is missing (causing the call to validateJSONString to fail)?

Reply:
Thanks Ken for catching that! I just added it.